### PR TITLE
LPD-37869 change friendlyUrlToggle locator for href

### PR DESCRIPTION
**Pull request by gergelyszalay**
Hi,

May I ask if you could review this PR? it is fixing test failures for journal-web/journalArticle.spec.ts.

https://liferay.atlassian.net/browse/LPD-37800

image

Thank you in advance!

### CODE REVIEW

**Comment by gergelyszalay** [comment]
ci:test:content-management-playwright

**Comment by gergelyszalay** [comment]
image
image

**Comment by gergelyszalay** [comment]
image
image
image

**Comment by gergelyszalay** [comment]
image

File: modules/test/playwright/tests/journal-web/pages/JournalEditArticlePage.ts [hunk 1]
```diff
@@ -265,7 +265,7 @@ export class JournalEditArticlePage {
 		await this.fillTitle(title);
 
 		if (!(await this.page.getByText('Never Expire').isVisible())) {
-			await this.page.getByRole('link', {name: 'Schedule'}).click();
+			await this.page.getByRole('button', {name: 'Schedule'}).click();
 		}
```
Comment (boton) [inline modules/test/playwright/tests/journal-web/pages/JournalEditArticlePage.ts:275]: Use the new util [playwright/utils/openFieldset.ts](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/utils/openFieldset.ts) for form and sidebar expandable panels.

You can view it in action [here](https://github.com/liferay-content-management/liferay-portal/pull/5924/commits/e6fa0e80e1faa4bf69622d1a7dc046e7ac2ebf9a) and [here](https://github.com/liferay-echo/liferay-portal/pull/16340/commits/f86c7c5479f075656e013bdde56fa7b494a33ccc).

File: modules/test/playwright/tests/journal-web/journalArticle.spec.ts [hunk 2]
```diff
@@ -1134,7 +1134,7 @@ translationTest(
 		});
 
 		if (await textBox.isHidden()) {
```
Comment (boton) [inline modules/test/playwright/tests/journal-web/journalArticle.spec.ts:1136]: I'm not sure if we can apply the new util [playwright/utils/openFieldset.ts](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/utils/openFieldset.ts) here.

Please, can you check?

**Review by boton — APPROVED** [review]
LGTM!
Nice job here @gergelyszalay ❤️ 

I only left two comments to try to use the new util for protect our test from markup updates.

> [!NOTE]
> Some flaky tests need to be fixed, but it's better to do them in separate issues.

image

File: modules/test/playwright/tests/journal-web/pages/JournalEditArticlePage.ts (hunk 1 above)
Comment (gergelyszalay) [reply modules/test/playwright/tests/journal-web/pages/JournalEditArticlePage.ts]: changed both for openFieldset! Thx @boton

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:content-management-playwright

**Review by boton — APPROVED** [review]

**Comment by boton** [comment]
ci:forward

**Pull request closed without merging**